apps/wallet-mobile/src/yoroi-wallets/cardano/common/signatureUtils.ts

```diff
@@ -1,10 +1,20 @@
-import {PrivateKey} from '@emurgo/cross-csl-core'
+import {SignTransactionRequest} from '@cardano-foundation/ledgerjs-hw-app-cardano'
+import * as CSL_TYPES from '@emurgo/cross-csl-core'
+import {Bip32PublicKey, PrivateKey} from '@emurgo/cross-csl-core'
+import {Addressing, createLedgerPlutusPayload} from '@emurgo/yoroi-lib'
 import blake2b from 'blake2b'
```

Comment:
blake2b library was removed from yoroi-lib recently because it caused issues

https://github.com/Emurgo/yoroi-lib/commit/d9f5e8c8952653b90f942b068dbbdfeb5f0bab7d

Reply:
https://github.com/Emurgo/yoroi-lib/commit/d9f5e8c8952653b90f942b068dbbdfeb5f0bab7d

Reply:
I'm not sure about the level of involvement of that library or other dependency but I think it's interesting to take it into account.